### DISCUSSION TITLE
Expedition 2 Beachhead: Phase 4 Find the Atlas

### DISCUSSION THREAD

**Discussion by LoverBoyUSA** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#37] [Q&A] [answered]
I have been to all the rendezvous and Polo cant locate any of the Atlas stations. I looked up on reddit but the rendezvous have changed.

**Comment by cwmonkey** [comment, marked as answer]
Ask him for black holes and he'll give you Atlas stations as well.

**Comment by LoverBoyUSA** [comment]
Yep i was just about to post that lol thanks tho
